### ISSUE TITLE
Failed to build without a11y

### ISSUE THREAD

**Issue by oil653** [pop-os/libcosmic#1088]
I'm trying to make a minimal shell, so I disabled the default features, and it failed to compile the lib.
I get the following compilation error: 
```
error[E0061]: this function takes 3 arguments but 4 arguments were supplied
   --> /home/kinx/.cargo/git/checkouts/libcosmic-41009aea1d72760b/b9c24d2/src/widget/spin_button.rs:246:28
    |
246 |     let decrement_button = make_button(
    |                            ^^^^^^^^^^^
...
249 |         [&spin_button.label, " decrease"].concat(),
    |         ------------------------------------------ unexpected argument #3 of type `std::string::String`
    |
note: function defined here
   --> /home/kinx/.cargo/git/checkouts/libcosmic-41009aea1d72760b/b9c24d2/src/widget/spin_button.rs:180:4
    |
180 | fn make_button<'a, T, Message>(
    |    ^^^^^^^^^^^
help: remove the extra argument
    |
248 -         "list-remove-symbolic",
249 -         [&spin_button.label, " decrease"].concat(),
248 +         "list-remove-symbolic",
    |

error[E0061]: this function takes 3 arguments but 4 arguments were supplied
   --> /home/kinx/.cargo/git/checkouts/libcosmic-41009aea1d72760b/b9c24d2/src/widget/spin_button.rs:252:28
    |
252 |     let increment_button = make_button(
    |                            ^^^^^^^^^^^
...
255 |         [&spin_button.label, " increase"].concat(),
    |         ------------------------------------------ unexpected argument #3 of type `std::string::String`
    |
note: function defined here
   --> /home/kinx/.cargo/git/checkouts/libcosmic-41009aea1d72760b/b9c24d2/src/widget/spin_button.rs:180:4
    |
180 | fn make_button<'a, T, Message>(
    |    ^^^^^^^^^^^
help: remove the extra argument
    |
254 -         "list-add-symbolic",
255 -         [&spin_button.label, " increase"].concat(),
254 +         "list-add-symbolic",
    |

For more information about this error, try `rustc --explain E0061`.
error: could not compile `libcosmic` (lib) due to 2 previous errors
```

I tried brute-forcing which module is it, and found it was a11y. 
After adding a11y back, it compiled without an issue.
